projects/media-viewer/src/lib/directives/drag-scroll.directive.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { DragScrollDirective } from './drag-scroll.directive';
+
+describe('DragScrollDirective', () => {
+  it('should create an instance', () => {
+    const directive = new DragScrollDirective();
```

Comment:
this test doesn't do much. if you want to test the setup, create it with the testbed

Reply:
i havent done any test yet, first code commit then test commit